js/components/user/list.vue

```diff
@@ -58,6 +58,13 @@ export default {
                 align: 'center',
                 type: 'metric',
                 width: 95
+            }, {
+                label: this._('Roles'),
+                key: 'roles',
+                sort: 'roles',
```

Comment:
For me, in production, a user has either one role (admin) or none.
With the sort, we can, with one clic on column header, view all the users with the admin privilege.
With the filtering, you would have to type explicitly 'admin' in the search control.
Sorting also allows you to group the users having the same roles.